### PR TITLE
Add in some information about the leaderboard renderers

### PR DESCRIPTION
**Pull request by thomasyu888**
#620

### CODE REVIEW

File: articles/evaluation_queues.md [hunk 1]
```diff
@@ -180,6 +180,15 @@ Clicking **Refresh Columns** will add these default columns.
 
 <img style="width: 80%;" src="/assets/images/leaderboard_columns.png">
 
+#### Leaderboard renderers
+
+* userid: Renders any Synapse user id to a Synapse user profile
+* date/epochdate: Renders epochtime in milliseconds to local time
+* markdown link: Support link in markdown format `[my link](url.com)`
```
Comment (thomasyu888) [inline articles/evaluation_queues.md]:
```suggestion
* markdown link: Support Synapse wiki markdown format`
```

**Review by kdaily — CHANGES_REQUESTED** [review]
date/epochdate should be defined separately. See PR https://github.com/thomasyu888/synapseDocs/pull/2 for other changes!

**Comment by thomasyu888** [comment]
The issue of date/epochdate is filed in a jira. They are duplicated

**Comment by kdaily** [comment]
https://sagebionetworks.jira.com/browse/SWC-4972 for reference. I would suggest we only document one (`date`) and say the other one is deprecated. Then file a separate issue to remove `epochdate` from the docs page when it's removed.

File: articles/evaluation_queues.md [hunk 2]
```diff
@@ -180,6 +180,15 @@ Clicking **Refresh Columns** will add these default columns.
 
 <img style="width: 80%;" src="/assets/images/leaderboard_columns.png">
 
+#### Leaderboard renderers
+
+* userid: Renders any Synapse user id to a Synapse user profile
+* date/epochdate: Renders epochtime in milliseconds to local time
```
Comment (jay-hodgson) [inline articles/evaluation_queues.md]: Note that in the web client, the user can select to show dates in UTC (in user settings) rather than local time.

**Review by kdaily — APPROVED** [review]

**Pull request merged**